smarc_bringup/launch/auv_scenarios.launch

```diff
@@ -0,0 +1,43 @@
+<launch>
+
+
+	<!-- Mode flags -->
+	<arg name="auv_name" default="lolo_auv"/>
+	<arg name="navigation_on" value="false"/>
+	<arg name="gazebo_env" value="ekf_localization_tests"/>
+
+	<!-- Initial position of the AUV -->
+	<arg name="x" value="0.0"/>
+	<arg name="y" value="10.0"/>
+	<arg name="z" value="-90.0"/>
+	<arg name="yaw" value="0.0"/>
+	
+	<!-- Underwater scenario for Gazebo -->
+	<include file="$(find smarc_worlds)/launch/$(arg gazebo_env).launch"/>
+
+	<!-- Launch AUV -->
+	<include file="$(find smarc_auvs)/models/$(arg auv_name)/launch/upload_$(arg auv_name).launch" pass_all_args="true"/>
+
+	<!-- Publishes: - tf world to odom based on initial pose of the AUV 
+					- Ground truth pose in odom frame
+					- if navigation_on = false, it also publishes the odom tf -->
+    <node pkg="odom_tf_bc" name="odom_tf_bc_node" type="odom_tf_bc_node" output="screen">
+		<param name="gt_topic" value="/$(arg auv_name)/pose_gt"/>
+		<param name="parent_frame" value="world"/>
+		<param name="child_frame" value="odom"/>
+		<param name="base_frame" value="$(arg auv_name)/base_link"/>
+		<param name="bc_odom_tf" value="$(arg navigation_on)"/>
+		<param name="gt_odom_topic" value="gt_in_odom"/>
+    </node>
+
+
+	<!-- Control components -->
+	<node pkg="smarc_keyboard_teleop" type="$(arg auv_name)_teleop.py" name="$(arg auv_name)_teleop" output="screen"/>
+	
+	<!-- Navigation components -->
+  	<include if="$(arg navigation_on)" file="$(find auv_ekf_localization)/launch/ekf_localization.launch">
+	</include>
+
+	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find smarc_bringup)/rviz/auv_rviz.rviz"/>
```

Comment:
Please remove this at least. Rviz should be launched on its own since it frequently needs to be relaunched.

Reply:
true, but it can be relaunched manually and I thought it's nice to load a config file for it